### PR TITLE
Girazoky try 5.0.6 beta

### PR DESCRIPTION
**Pull request by girazoki**

### CODE REVIEW

File: test/moonwall.config.json [hunk 1]
```diff
@@ -8,15 +8,18 @@
...
+            "reporters": ["basic", "html", "json"],
+            "reportFile": { "json": "./tmp/testResults.json" },
             "foundation": {
                 "type": "dev",
                 "launchSpec": [
                     {
                         "name": "tanssi",
                         "binPath": "../target/release/tanssi-node",
-                        "options": ["--dev", "--sealing=manual", "--no-hardware-benchmarks"],
+                        "options": ["--dev", "--sealing=manual", "--no-hardware-benchmarks", "--tmp"],
```
Comment (tmpolaczyk) [inline test/moonwall.config.json:22]: `--dev` already implies `--tmp`:

```
      --dev
          Specify the development chain. This flag sets `--chain=dev`, `--force-authoring`, `--rpc-cors=all`, `--alice`, and `--tmp` flags, unless explicitly overridden
```
```suggestion
                        "options": ["--dev", "--sealing=manual", "--no-hardware-benchmarks"],
```

File: test/moonwall.config.json [hunk 2]
```diff
@@ -8,15 +8,18 @@
             "name": "dev_tanssi",
             "envVars": ["DEBUG_COLORS=1"],
             "testFileDir": ["suites/dev-tanssi", "suites/common"],
+            "timeout": 120000,
+            "include": ["**/*test*"],
             "multiThreads": true,
-            "reporters": ["basic"],
+            "reporters": ["basic", "html", "json"],
+            "reportFile": { "json": "./tmp/testResults.json" },
```
Comment (tmpolaczyk) [inline test/moonwall.config.json:15]: This makes it a bit harder to see test failures when running locally, can we revert it for now?

Comment (girazoki) [reply test/moonwall.config.json:15]: yes

**Comment by girazoki** [comment]
Closed until things are more stable

**Pull request closed without merging**